### PR TITLE
fix decimal bug

### PR DESCRIPTION
**Pull request by arejula27**

### CODE REVIEW

File: src/database/model/project.py [hunk 1]
```diff
@@ -27,12 +27,8 @@ class OrmProject(OrmAIResource):
     start_date: Mapped[datetime] = mapped_column(Date, nullable=True, default=None)
     end_date: Mapped[datetime] = mapped_column(Date, nullable=True, default=None)
     founded_under: Mapped[str] = mapped_column(String(250), nullable=True, default=None)
-    total_cost_euro: Mapped[float] = mapped_column(
-        Numeric(scale=10, precision=2), nullable=True, default=None
-    )
-    eu_contribution_euro: Mapped[float] = mapped_column(
-        Numeric(scale=10, precision=2), nullable=True, default=None
-    )
+    total_cost_euro: Mapped[float] = mapped_column(Numeric(20, 2), nullable=True, default=None)
```
Comment (josvandervelde) [inline src/database/model/project.py:30]: I think 20 might be a bit too much. The first value is `Precision, the total number of digits in the value, i.e. on both sides of the decimal point `. So now you can have values up to `10^18`, so a quintillion euro. I guess `10^10` should be enough, so a precision of 12.

Moreover, I think it's a bit better to use `Numeric(precision=12, scale=2)`, that makes it clearer what those values are.

Comment (arejula27) [reply src/database/model/project.py]: i read that decimals also count fir the number of digits

Comment (josvandervelde) [reply src/database/model/project.py]: You could've read that it my response :) That's why I said `precision=12` instead of `precision=10`

Comment (josvandervelde) [reply src/database/model/project.py]: I still think that using named parameters is better. So rather `Numeric(precision=12, scale=2)` than `Numeric(12, 2)`. It makes it easier to read the code.

Comment (arejula27) [reply src/database/model/project.py]: :S , i have changed it to 12

**Review by josvandervelde — APPROVED** [review]

**Pull request merged**